Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -780,10 +786,7 @@ UpdateFspConfig (
 
   if (GetBootMode() == BOOT_ON_FLASH_UPDATE) {
```

Comment:
Should this be removed as BiosLock is set to zero right after this code block?

Reply:
It is used to set BiosInterface instead of BiosLock.
After checked our code, we did set BiosInterface = 1 after this code. 
So it should be not required. will remove it.